apps/email/js/html_cache_restore.js

```diff
@@ -21,23 +66,143 @@ var startupCacheEventsSent = false;
  * Set by build process. Set as a global because it
  * is also used in html_cache.js.
  */
-var HTML_COOKIE_CACHE_VERSION = '2';
+var HTML_CACHE_VERSION = '2';
 
 /**
- * Max size of cookie cache segments. Set as a global because it is also used by
- * html_cache.js.
+ * A function callback is registered if the model needs to be loaded before
+ * final view determination can be done. This should be a rare event, but can
+ * happen, see localOnModelLoaded for more details. config.js looks for this
+ * callback.
  */
-var HTML_COOKIE_CACHE_MAX_SEGMENTS = 40;
-
-// Use a global to work around issue with
-// navigator.mozHasPendingMessage only returning
-// true to the first call made to it.
-window.htmlCacheRestorePendingMessage = [];
+var startupOnModelLoaded;
```

Comment:
I went ahead and converted all the global vars in this file to be referenced via `window.`. Fixed in comment commit: 7d8f41f19303f833552bc490fe6bf87e1349526a